templates/taxi/index.html

```diff
@@ -0,0 +1,9 @@
+{% extends "base.html" %}
+
+{% block content %}
+    <ul>
+        <li>Number of cars: {{ cars_count }}</li>
+        <li>Number of drivers {{ drivers_count }}</li>
+        <li>Number of manufacturer {{ manufacturers_count }}</li>
+    </ul>
+{% endblock %}
```

Comment:
please, add a blank line at the end of the file

Reply:
Of course, thanks